src/pages/CrossChain/SwapForm/index.tsx

```diff
@@ -183,7 +186,7 @@ export default function SwapForm() {
       if (!library || !squidInstance || !route || !inputAmount || !outputAmount || !currencyIn || !currencyOut) return
       setSwapState(state => ({ ...state, attemptingTxn: true }))
       onTracking(MIXPANEL_TYPE.CROSS_CHAIN_SWAP_CONFIRMED)
-      const tx = await squidInstance.executeRoute({
+      const tx: any = await squidInstance.executeRoute({
```

Comment:
Um what is `any` for? I noticed that `tx` has a type of `TransactionResponse`

Reply:
because squid sdk return sida type 
`await squidInstance.executeRoute()`
so I decide to use any :v 

<img width="754" alt="image" src="https://github.com/KyberNetwork/kyberswap-interface/assets/33005392/7d2fd108-95ce-4b75-870e-a135dcf09b35">

<img width="489" alt="image" src="https://github.com/KyberNetwork/kyberswap-interface/assets/33005392/0cd39dcf-e56c-4304-94b4-950253d97f89">